scripts/test-installers.py

```
@@ -0,0 +1,304 @@
+
+class InstallerTests(unittest.TestCase):
+    def setUp(self) -> None:
+        self.temporary_directory = tempfile.TemporaryDirectory()
+        self.root = Path(self.temporary_directory.name)
+        self.release_directory = self.root / VERSION
+        self.release_directory.mkdir()
+        self.expected_contents: dict[str, bytes] = {}
+        self._make_release()
+        handler = lambda *args, **kwargs: QuietHandler(
+            *args, directory=str(self.root), **kwargs
+        )
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `handler` as a `def`

(E731)